analyzer/src/funTest/assets/projects/synthetic/composer-expected-output-with-provide.yml

```diff
@@ -1,7 +1,7 @@
 ---
 project:
   id: "PhpComposer:ort:composer-test-project:0.1.0"
```

Comment:
Why is the type in this id not also changed to "Composer"?

Reply:
Because I forgot about it 😄